src/compaction_filter.rs

```diff
@@ -106,7 +108,7 @@ extern "C" fn filter_v2(
         let filter = &mut (*(filter as *mut CompactionFilterProxy)).filter;
         let key = slice::from_raw_parts(key, key_len);
         let value = slice::from_raw_parts(value, value_len);
-        match filter.filter_v2(level as usize, key, value, value_type) {
+        match filter.filter_v2(level as usize, key, seqno, value, value_type) {
```

Comment:
why still v2

Reply:
I don't want to change it from `filter_v2` to `filter_v3`. Or we can find a better name to replace `filter_v2`. Any suggestions?

Reply:
just `filter`?

Reply:
`filter` is already used. How about `filter` and `featured_filter`?

Reply:
have no idea, stick to `v2` okay for me